sea-orm-codegen/src/entity/writer.rs

```diff
@@ -1667,6 +1706,29 @@ mod tests {
                 })
                 .to_string()
             );
+            assert_eq!(
+            >>>>>>> 4e74dde0 (sea-orm-cli: allow skipping impl ActiveModelBehavior for generated entities)
```

Comment:
There are several merge artifacts throughout this PR that should be cleaned up beforehand - but it would be great to see this merged in soon!